sites/partners/src/lib/hooks.ts

```diff
@@ -528,26 +529,51 @@ export const useZipExport = (
       // eslint-disable-next-line @typescript-eslint/no-explicit-any
       let content: any
       if (isLottery) {
-        content = await lotteryService.lotteryResults(
-          { id: listingId, includeDemographics, timeZone: dayjs.tz.guess() },
-          { responseType: "arraybuffer" }
-        )
+        content = useSecurePathway
+          ? await lotteryService.lotteryResultsSecure({
+              id: listingId,
+              includeDemographics,
+              timeZone: dayjs.tz.guess(),
+            })
+          : await lotteryService.lotteryResults(
+              { id: listingId, includeDemographics, timeZone: dayjs.tz.guess() },
+              { responseType: "arraybuffer" }
+            )
       } else {
         if (isSpreadsheet) {
-          content = await applicationsService.listAsSpreadsheet(
-            { id: listingId, includeDemographics, timeZone: dayjs.tz.guess() },
-            { responseType: "arraybuffer" }
-          )
+          content = useSecurePathway
+            ? await applicationsService.listAsSpreadsheetSecure({
+                id: listingId,
+                includeDemographics,
+                timeZone: dayjs.tz.guess(),
+              })
+            : await applicationsService.listAsSpreadsheet(
+                { id: listingId, includeDemographics, timeZone: dayjs.tz.guess() },
+                { responseType: "arraybuffer" }
+              )
         } else {
-          content = await applicationsService.listAsCsv(
-            { id: listingId, includeDemographics, timeZone: dayjs.tz.guess() },
-            { responseType: "arraybuffer" }
-          )
+          content = useSecurePathway
+            ? await applicationsService.listAsCsvSecure({
+                id: listingId,
+                includeDemographics,
+                timeZone: dayjs.tz.guess(),
+              })
+            : await applicationsService.listAsCsv(
+                { id: listingId, includeDemographics, timeZone: dayjs.tz.guess() },
+                { responseType: "arraybuffer" }
+              )
         }
       }
 
-      const blob = new Blob([new Uint8Array(content)], { type: "application/zip" })
-      const url = window.URL.createObjectURL(blob)
+      let url: string
+
+      if (useSecurePathway) {
+        url = content
+      } else {
+        const blob = new Blob([new Uint8Array(content)], { type: "application/zip" })
```

Comment:
question: This might not be a big deal. But the file downloaded by the browser has a different name when it is downloaded the secure route vs the direct download. The date is appended on the direct download, but not when downloaded via the s3 bucket.

example:
secure - `applications_export_1745935234440.zip`
direct download - `applications_export_1745935234440.zip`

Reply:
nice catch! the name will be a little different (the rec from aws is to not use `-` in the file name and so it'll use `_` instead but otherwise the timestamp will now be present